README.md

```diff
@@ -1 +1,42 @@
 # Terraform Provider for Kentik synthetic tests
+
+## Requirements
+
+- [Go](https://golang.org/doc/install) >= 1.16
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.13
+
+## Install
+
+Build and install the plugin so that Terraform can find and use it:
+
+```bash
+make install
+```
+
+## Development
+
+If you wish to work on the provider, you'll first need [Go](http://www.golang.org) installed on your machine (see [Requirements](#requirements) above).
```

Comment:
This seems to be repetitive and somewhat superfluous. Go compiler is required to build the provider, so it is not additional requirement for development. What we should say here is that anybody who wants to contribute to development is welcome to provide pull requests, ideally staged from a cloned repo. The `Requirements` above should IMHO also mention Kentik golang SDK (minimal version ... after we merge synthetics support to it, that is), terraform SDK (minimal version). We may get hints here how to use the local_apiserver (e.g. how at additional test cases), but this may as well be in the golang SDK repo.